IN-CORE/incore-playbook
# Release/0.3.0 (#23)

* reorganize structure of the code

reorganize dataset types; add socioecnomics datasets to redux

include format in the logic

add options to select dataset types

add category

add pretty description for scioeconomic input types

ugly implementation

bug

rewrite redux part

connect everything

fix bug

fix the menu

fix onSave

shortend the select menu

form check

switch basename back

drop cge upload

* added cge results graph

made bars vertical

fixed typos

added props

fixed errors

moved components and removed from route

* Description component for cge data #2 (#3)

* add one table example

* add static description table

* add some route example

* fix typo

* #3 cge population dislocation tables (#8)

* #3 CGE and Dislocation table outputs

* #3 Added table results for HUA and dislocation

* #3: Added color o dislocation data cell

* Connect run analysis page with new workflow #7 (#9)

* hardcode heatmap

* switch to a layer that only has dislocated and valid guid

* add layers

* no need for frontend to plot heatmap; geoserver can generate that style

* update datawolf config

* key got messed up

* fix bug

* triggering work flow seems to work

* heatmap working now

* connect everything together

* fix order

* rename

* rewrite all the modal components

* rewrite the whole view result page so it looks nicer

* population dislocation is working

* remove hard coding of population dislocation

* add cge

* fix CGE barchart width

* everything working except some of the geoserver styling

* MCS styling not working; missing a layer for interdependent network

* update population total dislocation table

* add by housing type

* fix bug

* drop the hardcoded data

* rename the results

* rename the table

* rename

* round percentage to only 1 decimal

* change the path back

* work with the new json output

* tweak styling

* tweak styling

* Remove most ESlint style warnings (#11)

* Remove simple ESlint related warnings

* Remove syntactic ESlint related warnings

* WIP: Add GitHub action #4 (#5)

* add github action to test npm run, docker build and docker push

* add ci.yaml

* updated github action

* removed node 16

Co-authored-by: YONG WOOK KIM <ywkim@illinois.edu>

* Create a metrics component #14 (#20)

* remove dummy code

* add percentage of change in CGE

* enforce check before run analyses

* add entry for metrics

* tab done

* functionality works

* try to center tables

* find way to toggle between metrics table and normal table

* fix column header width

* tab fixed

* fix the popup title section

* enable download but did not implement that feature yet

* accidentally remove the style

* create new joplin-bldg-mcs style on geoserver and match that; name of the failure probability changed

* fix duplicate key in hua

* can't fix duplicate key in view result entry table yet

* 16 download button cge popdislocation (#22)

* #16: Add download functionality for CGE and Pop Dislocation

* reduced the amount of white space on top padding for CGE control

* revert the change to router base path

* #16: add execution id to file name

* bump version up

* update change log

* delete unused routes

Co-authored-by: Diego Calderon <diegoac2@illinois.edu>
Co-authored-by: Gowtham Naraharisetty <59935416+vnarah2@users.noreply.github.com>
Co-authored-by: Santiago Nunez-Corrales <nunezco2@illinois.edu>
Co-authored-by: YONG WOOK KIM <ywkim@illinois.edu>
